pfs.ipmu.jp/usage.rst

```diff
@@ -0,0 +1,326 @@
+For your first time, please follow notification email to change your password 
+from an initial one (randomly created). 
+
+You can edir your account information from `LDAP account manipulator service <https://pfs.ipmu.jp/ldap-manip/>`_, 
+such as password, your real name, institution, and photo. 
+Also you can view list of all accounts from 
+
+* `List of existing accounts <https://pfs.ipmu.jp/ldap-manip/view_all.cgi>`_
+* `photo directory <https://pfs.ipmu.jp/ldap-manip/view_allphoto.cgi>`_
+
+======
+ToC
+======
+
+When accessing to https://pfs.ipmu.jp/, you will get newest list of 
+contents in this server. 
+
+LDAP account manipulator
+  You can view your account setting, list of all avail accounts, and photo 
+  directory. 
+  Also, you can edit your account setting (real name, password, institution, 
+  photo, etc.) from this service.
+List of fuze telecons
+  also see `Fuze conference system`_
+Content sharing services
+  Available services:
+
+  * Document server for PFS
+  * Photo archive
+  * WebDAV Storage (see `WebDAV Storage`_)
+
+Issue tracker and ticketing system
+  Issue tracker system -- Bugzilla : 
+  for help refer `Bugzilla help page <http://www.bugzilla.org/docs/tip/en/html/>`_
+Temporal sharing services
+  Available services:
+
+  * Etherpad list : web-based collaborative real-time editor
+  * pastebin
+  * EtherCalc : online spreadsheet
+
+pfs.ipmu.jp internal maillist (ML)
+  web interface of mailman, and you will get list of avail lists. 
+  It depends on settings per each list, you can view registered members, 
+  view logs of past emails, and also register (or request to register) on 
+  each list. 
+Internal wiki (not official)
+  for pfs.ipmu.jp server administration and scratch. 
+Server status viewer
+  for system administration use, you can view system status graph.
+
+------
+Photo archive
+------
+
+Photo archive for pfs.ipmu.jp, 
+contact `system administrator <pfs@pfs.ipmu.jp>`_ to put new 
+set of photos, after uploading your phots to WebDAV. 
+
+
+------
+WebDAV Storage
+------
+
+You can upload files via clients supporting WebDAV protocol, like cadaver on 
+Linux and MacOS. 
+You can upload/store/publish any project related files to this space, 
+including temporal file exchange. 
+
+Please refer `a page in the internal wiki <https://pfs.ipmu.jp/wiki/System/webdav>`_
+for how to connect to WebDAV storage.
+
+======
+Landfill services
+======
+
+Some landfill instances would avail. 
+(Note: landfill will be used for some testing purpose, but not a real 
+operated service.) 
+
+
+Online meeting
+------
+
+======
+Zoom system
+======
+
+PFS project uses Zoom system for teleconference. 
+For accessing on-line, you will need to `install clients <https://zoom.us/download>`_.
+
+You can check `list of planned teleconferences <https://calendar.google.com/calendar/embed?src=su0pbsaull17etlj62tet5anm0%40group.calendar.google.com&ctz=Asia/Tokyo>`_.
+
+
+======
+Fuze conference system
```

Comment:
Read INFRA-143 and INFRA-84, and review in the next ticket.
maybe we'd better to remove section in some future.